package.json

```diff
@@ -47,7 +47,7 @@
     "@types/mocha": "^5.2.7",
     "chai": "^4.2.0",
     "firebase-admin": "^10.1.0",
-    "firebase-functions": "^3.22.0",
+    "firebase-functions": "^3.23.0",
```

Comment:
Should we update the peer-depdency? It looks like our package won't work unless you are using firebase-functions 3.23 or above now since we explicitly try to import the Expression type in v2/params.

Reply:
You're extremely right!!!

Reply:
That probably means we need to do a minor or major bump then... WDYT?